client/pages/readonly.js

```diff
@@ -11,14 +11,14 @@ function Readonly({ establishment, isGranted, project, options, schemaVersion, s
   return (
     <Fragment>
       {
-        isGranted && <h3 className="est-title">{establishment.name}</h3>
+        isGranted && <h3 className="est-title">{establishment && establishment.name}</h3>
```

Comment:
`establishment &&` probably wants pulling outside the `<h3>` so it only renders the tag if it exists.

Reply:
```suggestion
        establishment && isGranted && <h3 className="est-title">{establishment.name}</h3>
```